src/Console/ConfigurationResolver.php

```diff
@@ -998,7 +998,7 @@ private static function separatedContextLessInclude(string $path): ConfigInterfa
 
         // verify that the config has an instance of Config
         if (!$config instanceof ConfigInterface) {
-            throw new InvalidConfigurationException(\sprintf('The config file: "%s" does not return a "PhpCsFixer\ConfigInterface" instance. Got: "%s".', $path, \is_object($config) ? \get_class($config) : \gettype($config)));
```

Comment:
i think you can update all usages of `is_object`

Reply:
In many cases we add the value for scalar types, e.g. "string#foobar":

https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/1f752c2482b1d4323cf3e73cbaeb05a4c7d6af80/tests/Test/AbstractIntegrationCaseFactory.php#L95

Reply:
In some cases with additional `null` check:

https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/1f752c2482b1d4323cf3e73cbaeb05a4c7d6af80/src/Fixer/ClassNotation/ClassAttributesSeparationFixer.php#L247

in other cases with additional resource check:

https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/1f752c2482b1d4323cf3e73cbaeb05a4c7d6af80/src/Fixer/Phpdoc/PhpdocReturnSelfReferenceFixer.php#L178

Should we maybe add a helper method somewhere which handles all these cases?
And it could return nicer outputs like `got string "foobar"` instead of `got "string#foobar"`,

Reply:
Would it be ok to merge this one as-is and have a look to the unification another time?

Reply:
sure, feel free to create a ticket for that